### PR TITLE
Added new export functionality for the agendaitem-list of a meeting

### PR DESCRIPTION
**Pull request by phgross**
This change introduces a new sablon template `agendaitem_list_template` wich can be configured per commitee or committeecontainer. The new action `Download agendaitem list` is available on the meeting view, and only offer a download but not storing the file anywhere.
bildschirmfoto 2016-05-31 um 12 22 49

To place the `download agendaitem list` on a wise position, I adjusted the sidebar on the meeting view (styling in plonetheme.teamraum).

bildschirmfoto 2016-05-31 um 12 23 01

@deiferni

### CODE REVIEW

File: opengever/meeting/protocol.py [hunk 1]
```diff
@@ -150,3 +150,17 @@ def add_protocol_type(self):
...
+
+    def add_protocol_type(self):
+        self.data['protocol'] = {
+            'type': translate(
+                _(u'label_agendaitem_list', default=u'Agendaitem list'),
+                context=getRequest())
+        }
+
+    def add_settings(self):
+        """Do not add the start-page setting for excertps."""
```
Comment (deiferni) [inline opengever/meeting/protocol.py]: excertps -> excerpts

File: opengever/meeting/protocol.py [hunk 2]
```diff
@@ -150,3 +150,17 @@ def add_protocol_type(self):
     def add_settings(self):
         """Do not add the start-page setting for excertps."""
```
Comment (deiferni) [inline opengever/meeting/protocol.py]: Please fix the typo here as well 😉

File: opengever/meeting/tests/test_agendaitem_list.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                protocol_template=self.sablon_template,
+                excerpt_template=self.sablon_template,
+                agendaitem_list_template=self.sablon_template))
+
+        self.committee = create(Builder('committee')
+                                .within(self.container)
+                                .having(repository_folder=self.repository_folder))
+        self.committee_model = self.committee.load_model()
+
+    def test_default_template_is_configured_on_commitee_container(self):
```
Comment (deiferni) [inline opengever/meeting/tests/test_agendaitem_list.py]: commitee -> committee (multiple times)

File: opengever/meeting/tests/test_agendaitem_list.py [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+        meeting = create(Builder('meeting')
+                         .having(committee=self.committee_model)
+                         .link_with(self.meeting_dossier))
+
+        browser.login().open(meeting.get_url())
+        browser.css('.download-agendaitem-list-btn').first.click()
+
+        self.assertEqual(browser.url, meeting.get_url())
+        self.assertEqual(
```
Comment (deiferni) [inline opengever/meeting/tests/test_agendaitem_list.py]: better, assert against the whole list (everything but exactly that error message is unexpected)

```python
self.assertEqual(
  [u'There is no agendaitem list template configured, ...'],
  error_messages()
)
```

File: opengever/meeting/browser/meetings/configure.zcml [hunk 5]
```diff
@@ -62,6 +62,14 @@
 
   <browser:page
       for="opengever.meeting.interfaces.IMeetingWrapper"
+      name="agenda_item_list"
+      class=".agendaitem_list.DownloadAgendaItemList"
+      permission="zope2.View"
+      allowed_attributes="as_json"
```
Comment (deiferni) [inline opengever/meeting/browser/meetings/configure.zcml]: `\me` 💛

File: opengever/meeting/tests/test_agendaitem_list.py (hunk 4 above)
Comment (phgross) [inline opengever/meeting/tests/test_agendaitem_list.py]: @deiferni ist nicht möglich da in der Sitzungsansicht bereits Statusmessages "vorbereitet" sind, diese sind aber nicht ersichtlich (`display: none`), ich vermute für JS Meldungen.

**Comment by deiferni** [comment]
almost ready 👍

**Comment by phgross** [comment]
@deiferni aktualisiert.

**Comment by deiferni** [comment]
🚢

**Pull request merged**